CoastalResilienceNetwork/GeositeFramework
# Truncate Launchpad description text

This implements a simple truncation on the description elements in the launch pad. When the text elements are over a certain height, the element is resized, a '...' is shown at the end, and a toggle button is added.

![download 2](https://cloud.githubusercontent.com/assets/18290666/21155055/a7a19e6c-c13e-11e6-868f-791ccac4acea.png)
![download 3](https://cloud.githubusercontent.com/assets/18290666/21155054/a7a1af2e-c13e-11e6-954e-2fff88c3a647.png)


To test:
* clone this branch, bring up the environment, and open the app
* click on the sidebar item to open the launchpad
* check that the first item is truncated and has a "View more" button
* check that clicking the button toggle the text truncation
* check that the other items are not truncated

Connects #749
